literature/jerkiewicz_2009_effect_12309/jerkiewicz_2009_effect_12309_p1_f1a.yaml

```diff
@@ -0,0 +1,79 @@
+curation:
+  version:
+  process:
+  - role: curator
+    name: Tobias Schmider
+    orcid: https://orcid.org/0000-0002-2911-2925
+    date: 2023-03-18
+experimental:
+  version: 1
+  instrumentation:
+    - type: potentiostat-galvanostat
+      model: EG&G Model 263A
+    - type: universal programmer
+      model: EG&G PAR model 175
+source:
+  version: 1
+  citation key: Jerkiewicz_2009_effect_12309
+  url: https://doi.org/10.1021/jp900478u
+  techniques:
+  - CV
+system:
+  version: 1
+  type: electrochemical
+  electrolyte:
+    type: aqueous
+    components:
+    - name: water
+      type: solvent
+      purity:
+        grade: nanopure
+        total ionic conductivity:
+          value: 18
+          unit: MOhm cm
+    - name: H2SO4
+      type: acid
+      concentration:
+        value: 0.05
+        unit: mol / l
+      purity:
+        grade: BDH Aristar
+    - name: CuSO4*H2O
+      type: salt
+      concentration:
+        value: 5
+        unit: mmol / l
+      source:
+        supplier: Aldrich
+      purity:
+        grade: 5N
+    - name: Ar
+      type: gas
+      purity:
+        grade: high-purity
+    temperature:
+      value: 298
+      unit: K
+  electrodes:
+    reference electrode:
+      type: RHE
+      description: H2 gas was bubbled through the reference electrode compartment in which the Pt/Pt-black electrode was immersed
+    counter electrode:
+      material: Pt
+      shape: gauze
+      crystallographic orientation: poly
+    working: electrode
```

Comment:
```suggestion
    working electrode:
```

Reply:
Can you verify that this is also not wrong in the old files that I reviewed?
I do not understand why this even passed the build. :)